src/fhir/guides/ig-atf/Home/Einfuehrung.page.md

```diff
@@ -1,41 +1,76 @@
+Diese Spezifikation definiert den übergeordneten Container (**MessageBundle**) und einen festen Bestandteil dieses Containers, den **MessageHeader**. Während spezifische Anwendungsfälle zusätzliche Einschränkungen oder Anforderungen an das Bundle und den MessageHeader haben können, bleibt das grundlegende Kommunikationsmuster unverändert.
 
 Jede Nachricht besteht also aus einem Bundle, welches mindestens den MessageHeader und je nach Anwendungsfall weitere Ressourcen enthält.
 
-{{render:src/fhir/guides/ig-atf/images/ATF-Scope.png}}
+## Stufenweise Einführung des App-Transport-Frameworks
+
+Die Implementierung des App-Transport-Frameworks erfolgt schrittweise, um den reibungslosen Übergang und die schrittweise Optimierung der Kommunikation im medizinischen Umfeld zu gewährleisten.
+
+### Stufe 1: Standardisierung und Zuverlässigkeit
+
+Die erste Stufe des App-Transport-Frameworks (ATF) etabliert wichtige Grundlagen für die strukturierte und zuverlässige Kommunikation im medizinischen Bereich. Sie führt folgende wesentliche Verbesserungen ein:
+
+- **Standardisierung und Klarheit**: Die klare **Use-Case-Definition** über das Codesystem des Anwendungsfalls sorgt für eine präzise und einheitliche Struktur beim Austausch von medizinischen Daten. Dies ermöglicht eine nahtlose Interoperabilität zwischen verschiedenen Systemen und Anwendungen, indem jeder Anwendungsfall klar definiert wird.
+
+- **Nachvollziehbarkeit und Verantwortlichkeit**: Durch die **eindeutige Benennung des Absenders und des implementierenden Systems** wird die Transparenz im Nachrichtenaustausch erhöht. Dies schafft eine klare Identifikation aller Teilnehmer und fördert die Sicherheit sowie die Verantwortlichkeit im Kommunikationsprozess.
+
+- **Sicherheit und Effizienz**: Die Einführung einer automatisierten **Empfangsbestätigung** gewährleistet, dass jede Nachricht korrekt empfangen und verarbeitet wurde. Der Absender erhält Rückmeldung über das Ergebnis der Verarbeitung, was die Effizienz des Kommunikationsprozesses verbessert und Fehler schnell identifiziert.
+
+- **Fehlertoleranz**: Durch das **Fallback-Szenario mit XSLT-Stylesheets** können Nachrichten bei Bedarf als menschenlesbare PDF-Dokumente übermittelt werden, falls technische Probleme die maschinelle Verarbeitung verhindern. Dies erhöht die Ausfallsicherheit und stellt sicher, dass der Inhalt der Nachricht dennoch zugänglich bleibt.
+
+**Zusammenfassung**: Stufe 1 legt die Grundlage für eine strukturierte, transparente und fehlertolerante Kommunikation im medizinischen Umfeld. Standardisierte Use Cases, klare Benennung der Systeme, automatisierte Empfangsbestätigungen und Fallback-Mechanismen verbessern die Sicherheit und Effizienz des Nachrichtenaustauschs erheblich.
+
+### Outlook: Automatisierter Use Case-Abgleich
```

Comment:
Ah hier ist Stufe 2, ich würde die schon ausdefinieren, oder?

Reply:
Überschrift lautet jetzt ### Outlook Stufe 2: Automatisierter Use Case-Abgleich

Reply:
Was denkst du, sollten wir lieber "preview" statt outlook schreiben? Beim ersten überfliegen kam mir das E-Mail Programm in den Sinn 😄